.gitignore

```diff
@@ -1,61 +0,0 @@
-*.a
-*.adb
-*.asm
-*.bin
-*.d
-!*.d/
-*.dSYM
-*.elf
-*.exe
-*.gcno
-*.gcda
-*.hex
-*.i
-*.ihx
-*.lib
-*.lod
-*.lst
-*.map
-*.o
-*.obj
-*.profraw
-*.rel
-*.src
-*.srec
-*.swp
-*.sym
-*.wsp
-*.ddc
-*.dds
-*.su
-*~
-.depend
-/.config
-/.config.*
-/.config-*
-/.config\ *
-/.cproject
-/.gdbinit
-/.project
-/.version
-/_SAVED_APPS_config
-/cscope.*
-/defconfig
-/Make.defs
-/nuttx
-/nuttx-*
-/nuttx.*
-/staging
-/tags
-/TAGS
-core
-Make*.dep
-uImage
-/external
-# $(TOPDIR)/Makefile.[unix|win]::$(CONTEXTDIRS_DEPS)
-.context
-.context\ *
-# $(TOPDIR)/Makefile.[unix|win]::$(DIRLINKS_EXTERNAL_DIRS)
```

Comment:
Wow. So many removals here

Reply:
I just deleted the .gitignore from this branch thinking that was the "right' way to address your comment?

Reply:
No. Please restore it to its original value, so it should not be a part of this PR at all

Reply:
Sorry to be dumb....I obviously added files to ignore myself that then became part of this PR. I understand that was stupid and I missed it. Because it was part of another commit I can't just revert the commit. So what is the best way to restore it to the original value?